.openpublishing.redirection.json

```diff
@@ -310,6 +310,16 @@
             "redirect_url": "/ef/core/modeling/inheritance",
             "redirect_document_id": false
         },
+        {
+            "source_path": "entity-framework/core/querying/async.md",
+            "redirect_url": "/ef/core/miscellaneous/async.md/",
+            "redirect_document_id": false
+        },
+        {
+            "source_path": "entity-framework/core/saving/async.md",
+            "redirect_url": "/ef/core/miscellaneous/async.md/",
```

Comment:
```suggestion
            "redirect_url": "/ef/core/miscellaneous/async",
```

Reply:
Sorry, somehow slipped out of the commit.